Cabal-tests/tests/UnitTests/Distribution/Simple/Glob.hs

```diff
@@ -101,13 +101,13 @@ testMatchesVersion version pat expected = do
   where
     isEqual = (==) `on` (sort . fmap (fmap normalise))
     checkPure globPat = do
-      let actual = mapMaybe (fileGlobMatches globPat) sampleFileNames
+      let actual = mapMaybe (\p -> (p <$) <$> fileGlobMatches version globPat p) sampleFileNames
```

Comment:
I am scratching my head a bit reading this.

Reply:
Fixed